packages/nimble-blazor/NimbleBlazor/wwwroot/NimbleBlazor.lib.module.js

```diff
@@ -50,6 +50,16 @@ export function afterStarted(Blazor) {
             };
         }
     });
+    // Used by NimbleBanner.razor
+    Blazor.registerCustomEventType('nimblebannertoggle', {
+        browserEventName: 'toggle',
+        createEventArgs: event => {
+            return {
+                oldState: !event.target.open,
+                newState: event.target.open
```

Comment:
The `event.details` has these properties in it, shouldn't need to calculate it again. Look at the nimblemenubuttontoggle above

Reply:
Changed.